spec/generate.php

```diff
@@ -355,16 +355,20 @@ function amqpTypeToLength($type, $e)
 $connectionContent .= "     */\n";
 $connectionContent .= "    private function onFrameReceived(AbstractFrame \$frame)\n";
 $connectionContent .= "    {\n";
-$connectionContent .= "        if (\$frame instanceof MethodFrame) {\n";
-$connectionContent .= "            if (\$frame instanceof MethodConnectionCloseFrame) {\n";
```

Comment:
I removed the `$frame instanceof MethodFrame` check because `MethodConnectionCloseFrame` implements it already.